tasks/skylab/LoomUtils.wdl

```diff
@@ -169,7 +169,7 @@ task AggregateSmartSeq2Loom {
         String? species
         String? organ
         String pipeline_version
-        String docker = "us.gcr.io/broad-gotc-prod/pytools:1.0.0-1661263730"
+        String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1686932671"
```

Comment:
Can you update the link to the script used in this task in the [multi-snSS2 Task table](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/README.md#multi-snss2-tasks-and-tools) (I think it's mentioned twice) and in the [outputs section](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/README.md#7-outputs) (1 mention)?

Reply:
Will do

Reply:
Hi Kaylee. I made the changes. Thanks!

Reply:
Thank you!